models/cow_protocol/ethereum/cow_protocol_ethereum_schema.yml

```diff
@@ -218,3 +218,37 @@ models:
         name: surplus_fee
         description: ""
       - *order_uid
+  - name: cow_protocol_ethereum_eth_flow_orders
+    meta:
+      blockchain: ethereum
+      project: cow_protocol
+      contributors: bh2smith
+    config:
+      tags: ['ethereum','cow_protocol','eth-flow', "orders"]
+    description: >
+      ETHFlow enables to sale of Native ETH via CoW Protocol. This works essentially by placing an (onchain) order, 
```

Comment:
nit: replace "to sale of Native ETH"  by "to sell Native ETH"

Reply:
```suggestion
      ETHFlow enables the sale of Native ETH via CoW Protocol. This works essentially by placing an (onchain) order, 
```